src/review/modules/components/AuthorizationNotice.js

```diff
@@ -0,0 +1,56 @@
+import React, { PropTypes } from 'react'
+import { connect } from 'react-redux'
+import { createStyleSheet } from 'jss-theme-reactor'
+import withStyles from 'material-ui/styles/withStyles'
```

Comment:
I have made an update to merge those two dependencies but it's not released yet.
```jsx
import { createStyleSheet, withStyles } from 'material-ui/styles'
```